platform/ios/src/MGLMapView.h

```diff
@@ -390,6 +390,13 @@ IB_DESIGNABLE
  */
 @property(nonatomic) CGFloat decelerationRate;
 
+/**
+ The gesture recognizer used to detect single taps on the map.
+ 
+ Your application can use this to modify the behavior of your own gesture recognizers.
+ */
+@property (nonatomic, readonly) UITapGestureRecognizer *singleTapGestureRecognizer;
```

Comment:
Yeah, let’s leave out the property for now. If someone really needs it, they can filter the array for UITapGestureRecognizers as a workaround. That leaves MGLMapView free to install additional gesture recognizers in the future for other purposes.